MAVProxy/tools/MAVExplorer.py

```diff
@@ -397,21 +391,7 @@ def cmd_fft(args):
 def save_graph(graphdef):
     '''save a graph as XML'''
     if graphdef.filename is None:
-        if 'HOME' in os.environ:
-            dname = os.path.join(os.environ['HOME'], '.mavproxy')
-            if os.path.exists(dname):
-                mp_util.mkdir_p(dname)
-                graphdef.filename = os.path.join(dname, 'mavgraphs.xml')
-        elif 'LOCALAPPDATA' in os.environ:
-            dname = os.path.join(os.environ['LOCALAPPDATA'], 'MAVProxy')
-            if os.path.exists(dname):
-                mp_util.mkdir_p(dname)
-                graphdef.filename = os.path.join(dname, 'mavgraphs.xml')
-        else:
-            graphdef.filename = 'mavgraphs.xml'
```

Comment:
This default has been lost.

I can see no conceivable reason this case should ever be taken.